tutorial/src/Constants.lagda

```diff
@@ -1,13 +1,13 @@
 % Constant Definitions
 
-Import List
+Import list
 ===========
 
 \begin{code}
 module Constants where
 
-open import Data.Bool using (Bool; true; false)
-open import Data.Nat using (ℕ; zero; suc)
+open import Sets.Enumerated using (Bool; true; false)
+open import Sets.Recursive using (ℕ; zero; suc)
```

Comment:
I do not either insist on using the Sets.Recursive and Sets.Enumerated modules.  That is just probably a residue of the original version that I started to work with, back in February.